modules/nw-operator-cr.adoc

```diff
@@ -292,8 +292,6 @@ endif::ibm-cloud[]
 |
 If your existing network infrastructure overlaps with the `100.64.0.0/16` IPv4 subnet, you can specify a different IP address range for internal use by OVN-Kubernetes. You must ensure that the IP address range does not overlap with any other subnet used by your {product-title} installation. The IP address range must be larger than the maximum number of nodes that can be added to the cluster.
 
-For example, if the `clusterNetwork.cidr` is `10.128.0.0/14` and the `clusterNetwork.hostPrefix` is `/23`, then the maximum number of nodes is `2^(23-14)=128`. An IP address is also required for the gateway, network, and broadcast addresses. Therefore the internal IP address range must be at least a `/24`.
```

Comment:
I think we just need to update `2^(23-14)=128` to `2^(23-14)=512`,  why removing all?

Reply:
yes, it makes sense to me